pkg/cmd/roachtest/tests/admission_control_death_spiral.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/option"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
+)
+
+func registerDeathSpiral(r registry.Registry) {
+	r.Add(registry.TestSpec{
+		Name:  "admission-control/death-spiral",
```

Comment:
Heh, evocative as it is, `admission-control/tpcc-severe-overload` is clearer. Could you fold this under `registerTPCCOverload`? It doesn't have to fit into the tpccOLAPSpec, but r.Add-ing another test in-line seems fine.